.github/workflows/release.yml

```diff
@@ -271,7 +271,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Determine version
-        run: echo "app-version=$(grep '@app_vsn "' ${{env.working-directory}}/mix.exs | cut -d '"' -f2)" > $GITHUB_ENV
+        run: echo "app-version=${GITHUB_REF#refs/tags/}" > $GITHUB_ENV
```

Comment:
Do you just want it to be the tag? GITHUB_REF will be something like `refs/tags/v0.62.0-rc.1` but GITHUB_REF_NAME will be `v0.62.0-rc.1`